### ISSUE TITLE
Printer offsets should perhaps be per-printer, rather than per-client

### ISSUE THREAD

**Issue by micahcowan** [kumoregdev/kumoreg#53]
I mean, ideally, the client-to-printer relationship is one-to-one anyway, in which cases this distinction is moot. But it's always possible a printer or two go down, and Brian _hasn't_ brought quite enough spare printers, in which case we've got a couple clients for the same printer, and any adjustments made to the offset in one client setting, have to be copied to the other client(s)' settings.

**Comment by jashort** [comment]
I don't expect it will be a major problem, but setting offsets per printer instead of per client would be convenient. I can't think of any reason to have two computers printing to the same printer with different offsets. (In other words, I expect weirdness to be confined to the printer, and the computers themselves don't actually do any layout or rendering.)

**Comment by micahcowan** [comment]
Yup my thoughts exactly. But clearly not worth bothering with at this point, safer to keep what we have, this close to con. We can consider implementing it later, for next year.
